gateway/src/main/java/io/zeebe/gateway/impl/configuration/GatewayCfg.java

```diff
@@ -24,6 +24,7 @@
   private NetworkCfg network = new NetworkCfg();
   private ClusterCfg cluster = new ClusterCfg();
   private ThreadsCfg threads = new ThreadsCfg();
+  private MonitoringCfg monitoring = new MonitoringCfg();
```

Comment:
For the broker configuration we have tests, maybe we need this also for the gateway? What do you think?

Reply:
Test are always good, please create a new issue for this, it's out of scope of this issue.

Reply:
Agree was just to clarify - > https://github.com/zeebe-io/zeebe/issues/2750